crates/hc_sandbox/tests/cli.rs

```diff
@@ -276,6 +279,97 @@ async fn generate_sandbox_memproof_deferred_and_call_list_dna() {
     assert!(exit_code.success());
 }
 
+/// Create a new default sandbox which should have DPKI enabled in the conductor config.
+#[tokio::test(flavor = "multi_thread")]
+async fn default_sandbox_has_dpki_enabled() {
+    clean_sandboxes().await;
+    package_fixture_if_not_packaged().await;
+
+    holochain_trace::test_run();
+    let mut cmd = get_sandbox_command();
+    cmd.env("RUST_BACKTRACE", "1")
+        .arg("--piped")
+        .arg("create")
+        .stdin(Stdio::piped())
+        .stdout(Stdio::piped())
+        .stderr(Stdio::inherit())
+        .kill_on_drop(true);
+
+    let mut sandbox_process = input_piped_password(&mut cmd).await;
+    let conductor_config = get_created_conductor_config(&mut sandbox_process).await;
+    assert_eq!(conductor_config.dpki.no_dpki, false);
```

Comment:
```suggestion
    assert!(!conductor_config.dpki.no_dpki);
```

Or is that equally confusing as the above? :)

Reply:
It's similar, but clippy enforces how you suggested.